modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_publisher/configuration/ordering.jsp

```diff
@@ -102,23 +102,28 @@ CPPublisherConfigurationDisplayContext cpPublisherConfigurationDisplayContext =
 </clay:row>
 
 <aui:script use="aui-base">
-	A.one('#<portlet:namespace />ordering').delegate(
-		'click',
-		(event) => {
-			var currentTarget = event.currentTarget;
-
-			var orderByTypeContainer = currentTarget.ancestor(
-				'.order-by-type-container'
-			);
-
-			orderByTypeContainer.all('.icon').toggleClass('hide');
-
-			var orderByTypeField = orderByTypeContainer.one('.order-by-type-field');
-
-			var newVal = orderByTypeField.val() === 'ASC' ? 'DESC' : 'ASC';
-
-			orderByTypeField.val(newVal);
-		},
-		'.icon'
-	);
+	var order = A.one('#<portlet:namespace />ordering');
+	if (order) {
```

Comment:
Ehi @MahmoudAzzam I don't think this is the right fix. What I understand from the previous version of the table (the one with aui:row) this was always working while this is occasionally working. 
I think you should focus on the generated id of the row, most probably the new tag is not prefixing the id value with the <portlet:namespace />...